apps/web/src/components/Layout/RouteErrorBoundary/RootErrorBoundary.tsx

```diff
@@ -0,0 +1,135 @@
+import { useIntl } from "react-intl";
+import { useLocation } from "react-router";
+import { ReactNode } from "react";
+import { tv } from "tailwind-variants";
+
+import { useTheme } from "@gc-digital-talent/theme";
+import { Button, Container, Flourish, Heading } from "@gc-digital-talent/ui";
+import { useLogger } from "@gc-digital-talent/logger";
+import { commonMessages, useLocale } from "@gc-digital-talent/i18n";
+
+import useErrorMessages from "~/hooks/useErrorMessages";
+import darkPug from "~/assets/img/404_pug_dark.webp";
+import lightPug from "~/assets/img/404_pug_light.webp";
+import {
+  GocLogoEn,
+  GocLogoFr,
+  GocLogoWhiteEn,
+  GocLogoWhiteFr,
+} from "~/components/Svg";
+
+const logo = tv({
+  base: "h-auto w-72",
+  variants: {
+    isDark: {
+      true: "hidden dark:block",
+      false: "block dark:hidden",
+    },
+  },
+});
+
+const reloadLink = (chunks: ReactNode) => (
+  <Button
+    mode="text"
+    className="font-bold"
+    onClick={() => window.location.reload()}
+  >
+    {chunks}
+  </Button>
+);
+
+export const RouteErrorBoundary = () => {
+  const intl = useIntl();
+  // eslint-disable-next-line no-restricted-syntax
+  const { locale } = useLocale();
+  const { mode } = useTheme();
+  const location = useLocation();
+  const error = useErrorMessages();
+  const logger = useLogger();
+
+  logger.notice(
```

Comment:
I'd suggest bumping this to "error" level.  I think we really want to know if this is hit, right?  Also, can you update the message to specify that it's the RootErrorBoundary?